style/javascript.md

```diff
@@ -552,3 +553,53 @@ Yes:
 ```js
 $(".some-class span").hide();
 ```
+
+#### Don't use Underscore
+
+We use ES6/7 which includes many of the features of Underscore.js! Using Underscore should be avoided in favor of these native language features.
+
+There are a couple of methods that are sufficiently complicated and don't have a direct equivalent so instead we have a [custom-built](https://lodash.com/custom-builds) copy of [lodash](https://lodash.com/) containing only those specific methods. You can find this file at: `third_party/javascript-khansrc/lodash/lodash.js` along with instructions on how to build it and exactly what methods are included.
+
+What follows is a method-by-method set of equivalents for what Underscore provides and what you could be using in ES6/7 instead:
+
+Method | Use...                                | ...instead of
+--------- | ------------------------------------- | ----------------------
+bind | `fn.bind(someObj, args)` | `_.bind(fn, someObj, args)`
+bind | `() => { ... }` | `_.bind(function() { ... }, this)`
+bindAll | `obj.method = obj.method.bind(someObj);` <sup>[1](#u1)</sup> | `_.bindAll(someObj, "method")`
+clone | No alternative at the moment! |
+debounce | Our custom lodash build. <sup>[2](#u2)</sup> |
+defer | `setTimeout(fn, 0);` | `_.defer(fn);`
+delay | `setTimeout(fn, 2000);` | `_.delay(fn, 2000);`
+each | `array.forEach(fn)` | `_.each(array, fn)`
+each | `for (const val of array) {}` | `_.each(array, fn)`
```

Comment:
Maybe to make the reason for two entries clearer, you could have the function column say something like `each [array]` and `each [obj]` (and similarly for several of the examples below).  Not sure if this is necessary or not.